getUnc_Unc.py

```diff
@@ -51,6 +52,7 @@ def parseOptions():
 
 if (not os.path.exists("plots") and doPlots):
     os.system("mkdir plots")
+    os.system('mkdir plots/'+opt.ERA)
```

Comment:
If I remember correctly, here the plotter is not working. We need to also fix this.